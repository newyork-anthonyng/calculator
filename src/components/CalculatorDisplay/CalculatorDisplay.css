.Calculator_Display {
	background-color: black;
	color: white;
	height: 100px;
	font-size: 80px;
	text-align: right;
	position: relative;
}

.Calculator_Display > span {
	position: absolute;
	bottom: 0;
	right: 0;
	text-align: right;
}
