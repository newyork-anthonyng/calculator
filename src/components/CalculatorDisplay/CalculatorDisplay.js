import React, { PropTypes } from 'react';
import './CalculatorDisplay.css';

const CalculatorDisplay = ({ text }) => (
	<div className="Calculator_Display">
		<span>{text}</span>
	</div>
);

CalculatorDisplay.propTypes = {
	text: PropTypes.string.isRequired
};

export default CalculatorDisplay;
