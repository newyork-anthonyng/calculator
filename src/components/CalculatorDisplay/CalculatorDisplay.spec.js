import React from 'react';
import renderer from 'react-test-renderer';
import CalculatorDisplay from './CalculatorDisplay';

describe('CalculatorDisplay', () => {
	it('should render', () => {
		const component = renderer.create(
			<CalculatorDisplay text="178" />
		);

		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
