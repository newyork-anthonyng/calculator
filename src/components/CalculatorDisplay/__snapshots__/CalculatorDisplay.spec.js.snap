exports[`CalculatorDisplay should render 1`] = `
<div
  className="Calculator_Display">
  <span>
    178
  </span>
</div>
`;
